site/topics/memory/esap-ram.rst

```diff
@@ -24,10 +24,39 @@ Address Register
 Including RAM in the System
 ===========================
 
+* Adding the RAM module to the system is trivial
```

Comment:
I'd probably play down _trivial_, maybe replace this line with something like _in order to add the module:_